miniwin/miniwin/src/include/miniwin_d3drmviewport_p.h

```diff
@@ -5,6 +5,7 @@
 #include "miniwin_d3drmobject_p.h"
 
 #include <SDL3/SDL.h>
+#include <span>
```

Comment:
I think this is unused.
```suggestion
```

 Perhaps you tried it for `PushVertices`?

Reply:
Yeah, it needed C++20